test/Succeed/Issue2284.warn

```diff
@@ -1,9 +1,11 @@
 Issue2284.agda:4,9-12
+warning: -W[no]ShadowingInTelescope
```

Comment:
Such warning should also be addressed by the user.
Maybe we have a classification of warnings into severity and `-Werror` levels for each one.